addons/account/report/account_tax.py

```diff
@@ -0,0 +1,131 @@
+# -*- coding: utf-8 -*-
+
+from odoo import api, models
+
+
+class ReportTax(models.AbstractModel):
+    _name = 'report.account.report_tax'
+
+    @api.model
+    def get_report_values(self, docids, data=None):
+        self.model = self.env.context.get('active_model')
+        docs = self.env[self.model].browse(self.env.context.get('active_ids', []))
+        return {
+            'doc_ids': self.ids,
+            'doc_model': self.model,
+            'data': data['form'],
+            'docs': docs,
+            'lines': self.get_lines(data.get('form')),
+        }
+
+    def _get_with_statement(self, user_types, options, domain=None):
```

Comment:
_get_with_statement() to remove (see my comment on the cash_basis/accrual option)